Epam.TestAutomation.Tests.DDT/Models/SkillsAndLocationsModel.cs

```diff
@@ -0,0 +1,18 @@
+using System.Text.Json.Serialization;
+
+namespace Epam.TestAutomation.Tests.DDT.Models;
+
+//Check that search results are related to the selected skill, country and city
+public class SkillsAndLocationsModel
+{
+    [JsonPropertyName("SearchPhrase")]
```

Comment:
Why did you create this class with all fields from json file and other classes with duplicate fields (e.g. SelectModel)?

Reply:
You can leave one class with all fields and remove others.